docs/Параметризация.md

```
@@ -57,15 +186,100 @@
+&ИсточникПараметров("ПолучитьМножители")
+Перем Множитель Экспорт;
+
+&МониторингПамяти
+Процедура ПриСозданииОбъекта()
+КонецПроцедуры
+
+&Бенчмарк
+&ИсточникПараметров("ПолучитьПараметрыБнечмарка")
+Процедура БенчмаркФакториала(Число, ТипАлгоритма) Экспорт
+
+	Если ТипАлгоритма = "Итеративный" Тогда
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)